addons/storyshots/storyshots-core/src/api/snapshotsTestsTemplate.js

```diff
@@ -2,11 +2,12 @@ import { describe, it } from 'global';
 import { addSerializer } from 'jest-specific-snapshot';
 
 function snapshotTest({ item, asyncJest, framework, testMethod, testMethodParams }) {
-  const { name } = item;
+  const { parameters, name } = item;
+  const displayName = (parameters && parameters.displayName) || name;
```

Comment:
Is there a test case that doesn't have a parameters.displayName attached? I didn't see one.

Reply:
I think this needs:

```
import { storyNameFromExport } from '@storybook/router/utils';

const displayName = (parameters && parameters.displayName) || storyNameFromExport(name);
```

If we want it to match what the user sees in Storybook. I'm guessing it'll work fine without.

Reply:
I just copypasted it from https://github.com/storybookjs/storybook/blob/next/lib/ui/src/components/sidebar/SidebarItem.tsx#L139

Reply:
Oh you might be right then. My bad!

Reply:
If you're getting it out of the story store then you don't need to do what I said. 👍

Reply:
It works as is, see https://github.com/storybookjs/storybook/pull/8000/commits/88863c112749cd6c80932248d381c7d30c9cbe38